.github/workflows/build-ffmpeg.yml

```diff
@@ -137,6 +208,7 @@ jobs:
             diffutils
             git
             make
+            mercurial
```

Comment:
What is using mercurial?

Reply:
I believe it's x265. Not sure what it does but it's optional.